src/generators/web/components/SearchBox/constants.mjs

```diff
@@ -0,0 +1,99 @@
+export const ORAMA_CONFIG = {
+  API_KEY: 'qopIuAERiWP2EZOpDjvczjws7WV40yrj',
```

Comment:
Ref https://github.com/nodejs/nodejs.org/blob/1d00bd8479b9238140c767b7be7101f39ce029bd/apps/site/next.constants.mjs#L194

Reply:
issue with this: it's will search on LTS version + nodejs.org
We need to use orama generator

Reply:
Unfortunately, it looks like `@orama/react-components` only works for the Orama Cloud, so we'd need to either use cloud orama, or write our own search bar.